codemark/scripts/cm_cli.py

```diff
@@ -81,6 +82,11 @@ def logout():
     """Logout the user"""
     codemark.logout.logout()
 
+@click.command()
+def signup():
+    '''Signs out the user'''
```

Comment:
Prefer using double quotes for docs purposes, and also minor typo : *signs ~~out~~ up the user*